packages/console/src/pages/Connectors/components/ConnectorForm/BasicForm.tsx

```diff
@@ -21,14 +22,14 @@ type Props = {
   isAllowEditTarget?: boolean;
   isDarkDefaultVisible?: boolean;
   isStandard?: boolean;
-  connectorType: ConnectorType;
```

Comment:
nit: seems `BasicForm` is only for social connectors, maybe can rename it as `SocialConnectorGeneralSettingForm` or similar to make the use cases more straightforward.